platform-tests/src/platform/acceptance/tls_version_test.go

```diff
@@ -19,6 +18,12 @@ var _ = Describe("Checking the TLS version", func() {
 		"system": func() string {
 			return testConfig.GetApiEndpoint() + ":443"
 		},
+		"doppler": func() string {
+			return "doppler." + GetConfigFromEnvironment("SYSTEM_DNS_ZONE_NAME") + ":443"
+		},
+		"concourse": func() string {
+			return "concourse." + GetConfigFromEnvironment("SYSTEM_DNS_ZONE_NAME") + ":443"
```

Comment:
This should be `deployer` instead of `coucourse`, otherwise it's testing the system router domain (due to the wildcard DNS)